reproserver/web/webcapture.py

```diff
@@ -75,6 +75,10 @@ async def post(self):
 class Dashboard(BaseHandler):
     @PROM_REQUESTS.sync('webcapture_dashboard')
     def get(self, upload_short_id):
+        self.post(upload_short_id=upload_short_id)
+
+    @PROM_REQUESTS.sync('webcapture_dashboard')
+    def post(self, upload_short_id):
```

Comment:
Why is this needed?

Reply:
to handle redirects from other pages

Reply:
What do you mean? And wouldn't a redirection use GET?

Reply:
If you take the breadcrumbs with `<a href="/web/">` , we have to change them to post with a form

Reply:
You didn't add GET, you added POST. What posts to that endpoint?

Reply:
this was due to previous design where we have a single form with two buttons and doing submit action, I think we don't need it anymore. removed this change in the latest patchset